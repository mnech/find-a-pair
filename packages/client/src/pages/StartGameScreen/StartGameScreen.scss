.start {
  display: flex;
  align-items: center;
  justify-content: center;
  height: 100vh;

  &__container {
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
    width: 50vw;
    height: 50vh;
    border: 1px solid black;
    border-radius: 10px;
  }

  &__title {
    max-width: 500px;
    color: var(--color-black);
    font-size: 18x;
    line-height: 24px;
    text-align: center;
    margin-bottom: 30px;
  }

  &__link {
    color: black;
    font-size: 16px;
    line-height: 22px;
    text-align: center;
    margin-bottom: 15px;

    &_game {
      color: blue;
      font-size: 22px;
      line-height: 26px;
      margin-bottom: 80px;
    }
  }
}
