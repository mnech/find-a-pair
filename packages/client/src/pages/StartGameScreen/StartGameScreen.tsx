import { Link } from 'react-router-dom';

import "./StartGameScreen.scss";

export const StartGameScreen = () => {
  return (
    <div className="start">
      <div className="start__container">
        <p className="start__title">Кликай по карточкам, запоминай, что под ними изображено, и находи пары одинаковых картинок</p>
          <Link to="/game" className='start__link'>
            Старт
          </Link>
        </div>
    </div>
  )
}
