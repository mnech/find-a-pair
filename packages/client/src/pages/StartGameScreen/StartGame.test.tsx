import { render } from '@testing-library/react';
import { StartGameScreen } from './StartGameScreen';
import { BrowserRouter as Router } from 'react-router-dom';

describe('Start game screen', () => {
  it('should render', () => {
    const element = render(
      <Router>
        <StartGameScreen />
      </Router>,
    );
    expect(element).toMatchSnapshot();
  });
});
