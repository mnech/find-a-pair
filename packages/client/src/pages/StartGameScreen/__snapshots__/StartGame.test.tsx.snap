// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Start game screen should render 1`] = `
<div>
  <div
    class="start"
  >
    <div
      class="start__container shadow-sm p-5 rounded"
    >
      <p
        class="start__title"
      >
        Кликай по карточкам, запоминай, что под ними изображено, и находи пары одинаковых картинок
      </p>
      <a
        class="start__link start__link_game"
        href="/game"
      >
        Старт
      </a>
      <a
        class="start__link"
        href="/leaderboard"
      >
        Лидерборд
      </a>
      <a
        class="start__link"
        href="/profile"
      >
        Профиль
      </a>
      <button
        class="btn btn-light btn btn-primary"
        type="button"
      >
        Выйти
      </button>
    </div>
  </div>
</div>
`;
